packages/smart-contracts/src/contracts/ProxyChainlinkConversionPath.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.5.0;
+
+import "./ChainlinkConversionPath.sol";
+import "./interfaces/ERC20FeeProxy.sol";
+
+
+/**
+ * @title ProxyChainlinkConversionPath
+ */
+contract ProxyChainlinkConversionPath {
+  using SafeMath for uint256;
+
+  address public paymentProxy;
```

Comment:
For future deployments: it would be nice to administrate that.

Reply:
added on JIRA